.clang-format

```diff
@@ -60,9 +60,6 @@ MacroBlockBegin: ''
 MacroBlockEnd:   ''
 MaxEmptyLinesToKeep: 1
 NamespaceIndentation: None
-ObjCBlockIndentWidth: 2
-ObjCSpaceAfterProperty: false
-ObjCSpaceBeforeProtocolList: false
```

Comment:
Do we actually want to remove those from here @qqaatw ? Isn't the newly added section below exclusively related to objective-C?

Reply:
Oh my bad, I completely missed that these are all `ObjC...` keys already anyway